docs/transform/modernize-userinterface-site-pages-api.md

```diff
@@ -726,8 +726,8 @@ Name|Description
 Name|Description
 :-----|:----------
 {Taxonomy Field}|The taxonomy field to update
-{'term ids split by |'}|List of term id's to set, multiple values can also be used when the taxonomy field is configured to accept multiple terms
-{'static boolean value'}|Static bool ('true', 'false') to indicate if the default term values have to be set in case the fiels already contains terms
+{'term ids split by &#124;'}|List of term id's to set, multiple values can also be used when the taxonomy field is configured to accept multiple terms
```

Comment:
So to expose a pipe, but not be a marker for a separate column, put a slash in front of it, like we do for the angle brackets.  So line 729 should look like:
`|{'term ids split by /|;'}|List of term id's to set, multiple values can also be used when the taxonomy field is configured to accept multiple terms|`